lib/widgets/item_details/main_info/item_main_info.widget.dart

```diff
@@ -60,6 +60,19 @@ class ItemMainInfoWidgetState extends BaseDestinyItemState<ItemMainInfoWidget> w
         ]));
   }
 
+  String getEnhancedDefinitionName(DestinyInventoryItemDefinition definition) {
+    String itemTypeDisplayName = definition?.itemTypeDisplayName ?? "";
+
+    if ((definition?.itemCategoryHashes?.indexOf(16) ?? -1) >= 0) {
```

Comment:
```suggestion
    final questCategoryHash = 16;
    if (definition?.itemCategoryHashes?.contains(questCategoryHash) ?? false) {
```

Just a small suggestion to make it easier to understand